### ISSUE TITLE
enable_queue=True throws exception from FastAPI

### ISSUE THREAD

**Issue by abaranovskis-redsamurai** [gradio-app/gradio#483]
**Describe the bug**
After upgrade to Gradio 2.7.5, enable_queue=True is causing exception, when Submit button is pressed. Same error when enable_queue=True is in interface or launch function

**To Reproduce**
1. pip install gradio==2.7.5
2. Set enable_queue=True in launch function
3. Go to Web page and press Submit, you should see an exception in the log

**Device information (please complete the following information):**
 - OS: macOS Monterey
 - Safari
 - 2.7.5 

**Additional context**
```
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/uvicorn/protocols/http/h11_impl.py", line 373, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/fastapi/applications.py", line 211, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/routing.py", line 656, in __call__
    await route.handle(scope, receive, send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/fastapi/routing.py", line 227, in app
    dependant=dependant, values=values, is_coroutine=is_coroutine
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/fastapi/routing.py", line 159, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/andrejb/infra/shared/katana-git/table-query-model/venv/lib/python3.7/site-packages/gradio/app.py", line 247, in queue_push
    data = body["data"]
KeyError: 'data'

```

**Comment by abidlabs** [comment]
Hmm I'm not able to reproduce this. For example, if I run the following code on colab or in a local jupyter notebook, it works feine. 

```python
import gradio as gr

gr.Interface(lambda x:x, "text", "text").launch(enable_queue=True)
```

Does this code work for you?

**Comment by abaranovskis-redsamurai** [comment]
I managed to reproduce it in Colab with your sample too.

Add examples array. On runtime, select one of the examples and click Submit, you should see error.

```
import gradio as gr

gr.Interface(lambda x:x, "text", "text", examples=['123']).launch(enable_queue=True)
```

Demo on [Colab](https://colab.research.google.com/drive/1hfGVjzZmDLRIHONsuJ30DwvVAq7XCq0i?usp=sharing)

**Comment by abidlabs** [comment]
Thanks for sharing that, very helpful. Looking into the issue

**Comment by abaranovskis-redsamurai** [comment]
I noticed selecting "examples" doesn't work on Hugging Face spaces even for previously deployed apps, probably on spaces, always the latest version of Gradio is used.

**Comment by abidlabs** [comment]
Should be fixed now as of `gradio 2.7.5.2`. Thanks so much for bringing this to our attention @abaranovskis-redsamurai!
